pkg/model/dev.go

```diff
@@ -695,6 +700,32 @@ func (dev *Dev) validate() error {
 	return nil
 }
 
+func (dev *Dev) validateSync() error {
+	for _, folder := range dev.Sync.Folders {
+		validPath, err := os.Stat(folder.LocalPath)
+
+		if err == nil {
+			if !validPath.IsDir() {
+				return oktetoError.UserError{
+					E:    fmt.Errorf("File paths are not supported on sync fields"),
+					Hint: "Update the `sync` field in your okteto manifest file to a valid directory path.",
+				}
+			}
+		} else if errors.Is(err, os.ErrNotExist) {
+			return oktetoError.UserError{
+				E:    fmt.Errorf("`stignore` path does not exist"),
```

Comment:
I think this can be any kind of path. It's not stignore path
```suggestion
				E:    fmt.Errorf("path '%s' does not exist", folder.LocalPath),
```